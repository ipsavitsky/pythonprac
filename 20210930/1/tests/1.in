1234,4124,645634