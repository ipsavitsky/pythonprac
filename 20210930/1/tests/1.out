[1234, 645634, 4124]
